src/artifacts.js

```diff
@@ -708,9 +708,10 @@ api.declare({
         tags: {taskId, runId, name},
       });
     } else {
-      let url = await this.s3Controller.generateGetUrl({
+      let url = await this.s3Controller.generateUrl({
```

Comment:
There is no method: `generateUrl`, or if there is I can't find the definition.

IMO, it be better to use `aws4` for URL signing here, instead of routing it via. `remotely-signed-s3`, because I have to cross reference to know what this does at HTTP level.